.github/workflows/ci.yml

```diff
@@ -51,10 +51,10 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v1
         with:
-          node-version: "14"
+          node-version: "16"
 
-      - name: Install project dependences
-        run: npm install -g prettier
+      - name: Install project dependencies
+        run: yarn global add prettier
```

Comment:
Minor: should we add a new GA job for linting the code (`./run-tests.sh --check-lint`) as described [here](https://github.com/reanahub/reana-ui/pull/213#discussion_r794635994)?